CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing
```

Comment:
Instead of adding this file, I think we could just add a link to the README pointing to our org's new contributor guidelines, available here: https://github.com/elixir-cloud-aai/elixir-cloud-aai/blob/dev/CONTRIBUTING.md

Reply:
https://github.com/elixir-cloud-aai/elixir-cloud-aai/blob/dev/resources/contributing_guidelines.md this is the correct link, right? This is a little specific.